### ISSUE TITLE
Inertia options param (callbacks) disrupting formkit error handling

### ISSUE THREAD

**Issue by jimmymueller** [GustavoFenilli/formkit-inertify#3]
**Describe the bug**
Anything passed to the inertia options param breaks expected formkit behavior. For example, the formkit handling of backend validation errors by field is disrupted when any inertia options are included.

**To Reproduce**
Add anything, even empty { }, to an inertia call. Only verified with inertia.post() so far.
I'm using this repo's adonisjs stack.

Example Playground.vue (I added console to vue global): 
```javascript
<template>
  <FormKit
    type="form"
    submit-label="Login"
    @submit="(fields, node) => node.context?.inertia.post(
      '/login',
      fields,
      // Form backend validation does not display for each field if ANY options param is set, even an empty {}
      // {}
      {
        onBefore: (visit, node) => {
          $console.log(visit, node);
          // Also it would be helpful to be able to return false from onBefore to cancel the request per the inertia docs
          // https://inertiajs.com/manual-visits#event-callbacks
        },
        onError: (errors, node) => {
          $console.log(errors);
          node.setErrors('testError', 'this should work too?'); //is this node being passed through correctly?
        }
      }
    )">
    <!-- <FormKit type="text" label="E-mail" name="email" validation="required|email" />
    <FormKit type="password" label="Password" name="password" validation="required" autocomplete /> -->
    <FormKit type="text" label="E-mail" name="email" />
    <FormKit type="password" label="Password" name="password" autocomplete />
  </FormKit>

  <pre>{{ $attrs }}</pre>
</template>
)">
```

**Versions (please complete the following information):**
 - FormKit: [e.g. 1.0.0-beta.10]
 - InertiaJS: [e.g. 0.11.0]
 - formkit-inertify: [2.0.0-alpha.4]

**Comment by GustavoFenilli** [comment]
Thanks for the report, I will take a look at the example.

**Comment by GustavoFenilli** [comment]
It should be fixed after I bundle and send to NPM ( parcel is not really behaving right now, so trying to fix it )

**Comment by jimmymueller** [comment]
I tried the new code in the adonisjs stack and seems to be working great so far! Once alpha5 is in NPM, we'll get it into our app and test in that environment.

**Comment by GustavoFenilli** [comment]
Published as 2.0.0-alpha.5
